scripts/package.sh

```diff
@@ -252,9 +249,15 @@ main() {
     fi
   fi
 
-  if [[ $appimage == true ]]; then source scripts/appimage.sh; fi
-  if [[ $bundle == true && $dmg == true ]]; then source scripts/appdmg.sh "${package_name}"; fi
-  if [[ $innosetup == true ]]; then source scripts/innosetup/innosetup.sh -b "${build_dir}"; fi
+  if [[ $appimage == true ]]; then
+    source scripts/appimage.sh $flags --static
```

Comment:
Does this make it build twice? Maybe we should specify `--nobuild`.

Reply:
The thing is that the first build makes some modifications to be able to execute lite-xl in a portable way, also the whole directory structure gets compacted into data, meanwhile an appimage is kind of a chroot with escalated privileges and the typical /usr/ directory tree is desired with regular lookup paths.

Reply:
Alright but this is a bit of a waste. Hopefully we can find a way to avoid doing this.